src/components/views/leftsidebar/subcomponents/TabsPanel.js

```diff
@@ -105,35 +130,45 @@ class PanelContent extends React.Component{
   }
 }
 
+PanelContent.propTypes = {
+	activeTab:React.PropTypes.object
+};
+
 class TabsPanel extends React.Component{
   constructor() {
     super();
     this.state = {
     panelData : null, 
      activeTab: null
-    }
-    this.handleClick = this.handleClick.bind(this)
+    };
+    this.handleClick = this.handleClick.bind(this);
   }
   
+  componentWillMount(){
```

Comment:
This is one of the React lifecycle's methods which are used in the component lifecycle.
Additional Information - [componentWillMount()](https://facebook.github.io/react/docs/react-component.html#componentwillmount). What it is doing is setting states of different states using the props the component receives. 
We probably need to look and accept an extensive styling guide not just for code but for documentation too.